Examples/DataRepresentation/Mesh/AutomaticMesh.cxx

```diff
@@ -100,12 +100,10 @@ main(int, char *[])
   //  Software Guide : EndLatex
 
   // Software Guide : BeginCodeSnippet
-  meshSource->AddTetrahedron(
-    meshSource->AddPoint( -1, -1, -1 ),
-    meshSource->AddPoint(  1,  1, -1 ),
-    meshSource->AddPoint(  1, -1,  1 ),
-    meshSource->AddPoint( -1,  1,  1 )
-  );
+  meshSource->AddTetrahedron( meshSource->AddPoint( -1, -1, -1 ),
+                              meshSource->AddPoint( 1, 1, -1 ),
+                              meshSource->AddPoint( 1, -1, 1 ),
+                              meshSource->AddPoint( -1, 1, 1 ) );
```

Comment:
I like the previous much better. A change of the name of the variable `meshSource` or the function `AddTetrahedron` should only affect line 103, not the following lines.

Reply:
This is an example of a `con` that is outweighed by the `pro`s of having a uniform, easily automated fixed format style.

Reply:
It seems that you can configure `AlignAfterOpenBracket: DontAlign` and then fixed `ContinuationIndentWidth` is used for alignment instead.

Reply:
@lassoan Thank you, I'll try that.